scripts/zephyr_module.py

```diff
@@ -247,7 +247,7 @@ def process_twister(module, meta):
     return out
 
 
-def process_meta(zephyr_base, west_projects, modules, extra_modules=None,
+def process_meta(zephyr_base, west_projs, modules, extra_modules=None,
```

Comment:
nit: why is variable renamed?

Reply:
easier to search for just the function `west_projects` or the variable `west_projs` now.

and we use `projs` in vars elsewhere in python code in Zephyr.

Reply:
To better differentiate the variable from the function